benchmarks/race-h.py

```diff
@@ -0,0 +1,56 @@
+"""
+RACE: Large-scale ReAding Comprehension Dataset From Examinations
+
+Evaluation on RACE-H, the subset of the dataset corresponding to high school examinations, is implemented
+
+Guokun Lai, Qizhe Xie, Hanxiao Liu, Yiming Yang, Eduard Hovy
+https://arxiv.org/abs/1704.04683
+
+# eval race-h
+inspect eval race-h.py
+"""
+
+from inspect_ai import Task, task
+from inspect_ai.dataset import Sample, hf_dataset
+from inspect_ai.model import GenerateConfig
+from inspect_ai.scorer import choice
+from inspect_ai.solver import multiple_choice
+
+
+def record_to_sample(record):
```

Comment:
We typically put the `record_to_sample()` function at the bottom after the task definition.

Reply:
Sure. Updated in recent commits.